lib/charms/loki_k8s/v0/loki.py

```diff
@@ -4,7 +4,7 @@
 #
 # Learn more at: https://juju.is/docs/sdk
 
-"""## Overview.
+r"""## Overview.
```

Comment:
Why the `r`? Interesting!

Reply:
Because we are using backslashes in the docstring:
```
lib/charms/loki_k8s/v0/loki.py:7:1: D301 Use r""" if any backslashes in a docstring
```